cyhy/db/ticket_manager.py

```diff
@@ -77,44 +77,50 @@ def __check_false_positive_expiration(self, ticket, time):
             if fp_expiration_date < time:
                 ticket["false_positive"] = False
                 event = {
-                    "time": time,
                     "action": TICKET_EVENT.CHANGED,
+                    "delta": [{"from": True, "to": False, "key": "false_positive"}],
                     "reason": "False positive expired",
                     "reference": None,
-                    "delta": [{"from": True, "to": False, "key": "false_positive"}],
+                    "time": time,
                 }
                 if self.__manual_scan:
                     event["manual"] = True
                 ticket["events"].append(event)
 
     def __generate_ticket_details(self, vuln, ticket, check_for_changes=True):
-        """generates the contents of the ticket's details field using NVD data.
+        """Generates the contents of the ticket's details field using NVD data.
         if check_for_changes is True, it will detect changes in the details,
```

Comment:
Since we're already editing this docstring the summary should be in the imperative mood and there should be a blank line after the summary.

```suggestion
        """Generate the contents of the ticket's details field using NVD data.

        If check_for_changes is True, it will detect changes in the details,
```